pkg/templates/machinecontroller/cloudprovider_specs.go

```diff
@@ -121,6 +121,7 @@ type AzureSpec struct {
 	RouteTableName    string            `json:"routeTableName"`
 	SecurityGroupName string            `json:"securityGroupName"`
 	Zones             []string          `json:"zones"`
+	ImagePlan         *AzureImagePlan   `json:"ImagePlan"`
```

Comment:
* This should eventually be `omitempty`, since it's a pointer, but there's no `omitempty` in the MC type, so I didn't add it here.
* The capital `I` in `ImagePlan` is expected because it's also like that in the MC type.

Reply:
ouch, that's a BUG in the MC

Reply:
MC 1.15.1 fixes this issue. I've updated the PR to use 1.15.1 instead of 1.15.0. PTAL.